migrations/src/postDeploymentSetup.js

```diff
@@ -44,21 +43,19 @@ module.exports = async () => {
   // Only transfer tokens in testnets
   if (NETWORK !== 'mainnet') {
     // Add new instance of StakeManager contract & Deployer address as Minter
-    await schellingCoin.addMinter(stakeManagerAddress);
+    const initialSupply = await schellingCoin.INITIAL_SUPPLY();
     await schellingCoin.addMinter(signers[0].address);
+    await schellingCoin.mint(stakeManagerAddress, initialSupply);
 
     if (SCHELLING_COIN_ADDRESS !== '') {
-      const { StakeManager: oldStakeManagerAddress } = await readOldDeploymentFile();
-
       // if previous instances of Schelling Coin is reused again and again,
       // then initial balance will get depleted, thus intial tokens minting is needed,
       // each time Schelling Coin instance is reused
       const initialSupply = await schellingCoin.INITIAL_SUPPLY();
       await schellingCoin.mint(signers[0].address, initialSupply);
+      await schellingCoin.mint(stakeManagerAddress, initialSupply);
```

Comment:
> await schellingCoin.mint(stakeManagerAddress, initialSupply);

We can remove this from here & use it after the `if` condition. @adi44

Reply:
Ok